voidcrew/code/modules/cargo/packs.dm

```diff
@@ -30,3 +30,39 @@
 		/obj/item/circuitboard/machine/shuttle/engine/electric
 	)
 	crate_name = "Shuttle in a Box"
+
+//////////////////////////////////////////////////////////////////////////////
+/////////////////////////// Cassette Stuff ///////////////////////////////////
```

Comment:
same here (i know every else has them but they're just dumb bloat
use 
```
/*
 * Cassette stuff
 */
``` 
if you REALLY think it's necessary to have a comment separating them, but i don't think its really that important